src/main/Tray.js

```diff
@@ -0,0 +1,63 @@
+import { app, Menu, Tray as ElectronTray } from 'electron';
+import path from 'path';
+import createWindow from './createWindow';
+
+export default class Tray {
```

Comment:
Does this need to be a whole object? `create()` is only called once and it doesn't need any state, so maybe just a `createTray` function that takes an `updateAttachedWindow` callback?

Reply:
I went for a exported `createTray()` function initially. However, since it also needs to manage the state of the window, I finally went for using a class.